src/apps/organisations/api.ts

```diff
@@ -1,18 +1,39 @@
 import { authenticatedInstance } from "../../api";
-import { Organisation } from './types';
+import { Organisation, EditableOrganisationFields } from './types';
 import { AxiosResponse } from "axios";
 
 const api = {
   getUserOrgs: (username:string) => authenticatedInstance.get(`/users/${username}/orgs/?sortDesc=lastUpdated`),
-  retrieve: (username:string, q: string = "") => authenticatedInstance.get(`/users/${username}/orgs/?limit=0`, {
+  retrieve: {
```

Comment:
What is this supposed to retrieve? All orgs? The orgs for a specific user? This should probably be modelled more closely to the apis for dictionaries and sources.